package indigo.pb;

import "sender_message.proto";

option optimize_for = LITE_RUNTIME;

option java_package = "ru.indigosystem.serv.pb";

option java_outer_classname = "GeoMessage";


message indigo_geo {
    required double longitude = 1;

    required double latitude = 2;

    required int32 unixtime = 3;

    optional int32 speed_kmh = 4 [ default = 0 ];

    optional int32 track = 5 [ default = 0 ];

    optional int32 satellites_used = 6 [ default = 0 ];

    extend indigo_msg {
	repeated indigo_geo geo_coords = 100;
    }

    extend internal_msg {
    	optional indigo_geo geo_coords_in = 100;
    }
}

